scripts/run-typewriter-clean-metadata-keep-annotations.ps1

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+
+# This script runs typewriter which results in: 1) metadata processed through an xslt and 2) documentation added as annotations in the metadata
+
+Push-Location -Path $env:TypewriterDirectory
+
+& chmod +x $env:TypewriterExecutable
```

Comment:
we already have a very similar PS1. Could we update the original one instead of duplicating it so it can be used in both contexts?

Reply:
Thank you, good call. Will change.